sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -279,6 +269,50 @@ case class Ceil(child: Expression) extends UnaryMathExpression(math.ceil, "CEIL"
   override protected def withNewChildInternal(newChild: Expression): Ceil = copy(child = newChild)
 }
 
+@ExpressionDescription(
+  usage = "_FUNC_(expr) - Returns the smallest integer not smaller than `expr`.",
+  examples = """
+    Examples:
+      > SELECT _FUNC_(-0.1);
+       0
+      > SELECT _FUNC_(5);
+       5
+      > SELECT _FUNC_(3.1411, 3);
+       3.142
+      > SELECT _FUNC_(3.1411, -3);
+       1000
+  """,
+  since = "3.3.0",
+  group = "math_funcs")
+object CeilExpressionBuilder extends ExpressionBuilder {
+  def build(expressions: Seq[Expression]): Expression = {
+    if (expressions.length == 1) Ceil(expressions.head)
+    else if (expressions.length == 2) {
+      val child = expressions(0)
+      val scale = expressions(1)
+      val scaleV = scale.eval(EmptyRow)
+      if (scaleV == null) throw new AnalysisException("Scale parameter can not be null")
+
+      (child.dataType, scaleV.asInstanceOf[Int]) match {
+        case (_, 0) => Ceil(child)
+        case (_, x) if x < 0 => RoundCeil(Cast(child, LongType), scale)
+        case (_: IntegralType | ShortType, _) => RoundCeil(Cast(child, LongType), scale)
```

Comment:
how about `ByteType`?

Reply:
Good question. In case of `ByteType` do we need to cast to `Long`/`Double` ?

Reply:
Let's cast ByteType to `LongType` ? I don't find any UT for `Round` with ByteType. Do you any specific case in mind, please ?

Reply:
casting to long SGTM

Reply:
Ok, i'll do it. I'm not sure about the UT cases though, could you give me few cases, please ? thanks !

Reply:
actually, you wrote `IntegralType` here, which covers byte/short/int/long already.

Reply:
ah ok, so let's remove ShortType and do this ?
```
(child.dataType, scaleV.asInstanceOf[Int]) match {
        case (_, 0) => Ceil(child)
        case (_, x) if x < 0 => RoundCeil(Cast(child, LongType), scale)
        case (_: IntegralType, _) => RoundCeil(Cast(child, LongType), scale)
        case (_ : FloatType, _) => RoundCeil(Cast(child, DoubleType), scale)
        case _ => RoundCeil(child, scale)
}
```